jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/xml/ServicesProcessDataEventListener.java

```diff
@@ -126,7 +130,24 @@ public void onNodeAdded(Node node) {
             } else {
                 processDescriptor.getReusableSubProcesses().add(processId);
             }
+        } else if (node instanceof TimerNode) {
+            TimerNode timerNode = (TimerNode) node;
+            processDescriptor.getTimers().add(new TimerDesc(timerNode.getTimer().getId(), timerNode.getId(), (String) timerNode.getMetaData().get("UniqueId"), timerNode.getName()));
+        } else if (node instanceof StartNode) {
+            StartNode startNode = (StartNode) node;
+            processDescriptor.getTimers().add(new TimerDesc(startNode.getTimer().getId(), startNode.getId(), (String) startNode.getMetaData().get("UniqueId"), startNode.getName()));
```

Comment:
Do we have to check if starNode.getTimer() could be null here?

Reply:
@nmirasch makes sense, thanks, fixed now.